tests/test_modeling_tf_bart.py

```diff
@@ -28,12 +30,16 @@
 if is_tf_available():
     import tensorflow as tf
 
-    from transformers import BartConfig, TFBartForConditionalGeneration, TFBartModel
-    from transformers.tokenization_bart import BartTokenizer
+    from transformers import TFBartForConditionalGeneration, TFBartModel
+    from transformers.modeling_tf_bart import TFSinusoidalPositionalEmbedding
 
 
 @require_tf
-class ModelTester:
+class TFBartModelTester:
```

Comment:
yaaay :-)

Reply:
on this PR you have to say TFBartyaay or people won't understand what's going on.